src/utils/synchronization.ts

```diff
@@ -428,7 +427,7 @@ export async function postData(
                     skipExistingCheck: false,
                     format: "json",
                     async: false,
-                    dryRun: false,
+                    dryRun: additionalParams?.dryRun ?? false,
```

Comment:
There's no need for this setting here, as in line 431 we're already destructuring ``additionalParams``. So ``dryRun: false`` here is the default behvaiour and if ``dryRun`` is set it will overwrite the default value in the destructuring.

Reply:
Done!, 
I didn't know that the order of destructuring and assigning fields affected to be default or assigned value, thanks.